apps/web/pages/[user]/[type].tsx

```diff
@@ -26,13 +26,15 @@ export default function Type({ slug, user, booking, away, isBrandingHidden, resc
         eventSlug={slug}
         rescheduleUid={rescheduleUid ?? undefined}
         hideBranding={isBrandingHidden}
+        org={org}
       />
       <Booker
         username={user}
         eventSlug={slug}
         bookingData={booking}
         isAway={away}
         hideBranding={isBrandingHidden}
+        org={org}
```

Comment:
Makes booker aware of org so that it can send the org back to fetch the event through public.event tRPC call.